torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -115,7 +115,7 @@ class Raft_Large_Weights(WeightsEnum):
         },
     )
 
-    DEFAULT = C_T_V2
+    DEFAULT = C_T_SKHT_K_V2
```

Comment:
I can't comment on the right spot of the code but you need to change this also below at the:
```
@handle_legacy_interface(weights=("pretrained", Raft_Large_Weights.C_T_SKHT_K_V2))
```